### PR TITLE
Add message to use the full image if there are no update files

### PR DESCRIPTION
**Pull request by Adimote**
This reverts commit 734756cdd01feeb971e4d21fffdbc2e74363eb35. (because I accidentally pushed the original change to master)

### CODE REVIEW

File: content/updates/_index.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ The update files are the easiest way to apply an update and are the recommended
 approach. The full images are provided in case an update goes wrong or you need
 to completely wipe your Pi for some reason.
 
+If no update files are provided, you should update using the [full image](/kit/pi/sd-card) download.
```
Comment (trickeydan) [inline content/updates/_index.md]: This should be "update file" as you are only talking about a singular file in this context.

Comment (Adimote) [reply content/updates/_index.md]: fixed in 33ec8a71acc0d4b1041b49e1612feb5353545513

**Comment by RealOrangeOne** [comment]
I suggest also adding this to the updates page

**Comment by PeterJCLaw** [comment]
This is rendered obsolete by #132.

**Comment by Adimote** [comment]
As #132 was merged, this is now obsolete

**Pull request closed without merging**